pkgs/development/lua-modules/overrides.nix

```diff
@@ -966,7 +966,8 @@ in
 
   rocks-dev-nvim = prev.rocks-dev-nvim.overrideAttrs (oa: {
 
-    doCheck = true;
+    # E5113: Error while calling lua chunk [...] pl.path requires LuaFileSystem
+    doCheck = luaOlder "5.2";
```

Comment:
Q: does the test failing on <5.2 imply the package is broken for those older versions?

Are we just "hiding" the issue by disabling the check phase, or do we know the issue is exclusive to running tests in the build sandbox?

Reply:
That's what I was curious about after 

>LGTM. I'll push a release with a rockspec that only supports Lua 5.1 (for the whole rocks.nvim suite of plugins).

I thought it was just our test system dependency propagation issue.

Reply:
I'm not familiar with this package, so I'll defer to your judgement.

If this package was untested (but apparently working) until recently, and then a checkPhase (incompatible with older versions) was recently added, then disabling it seems fine.

I guess the proof will be in whether users report issues with the older versions.

Reply:
This explains the issue: https://github.com/nvim-neorocks/lux/issues/715#issuecomment-2895873648

TL;DR: it's because the tests use Neovim as the Lua interpreter and busted tries to `require` a Lua library written in C that was compiled against an incompatible Lua version.